src/widgets/HomePageWidgets/Hero/model/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface IHero {
+  bgImage: string;
+  topic: string;
+  title: string;
+  description: string;
+}
+
+export interface IHeroData {
```

Comment:
better to rename it to IHeroSlide, because it is interface for HeroSlide component